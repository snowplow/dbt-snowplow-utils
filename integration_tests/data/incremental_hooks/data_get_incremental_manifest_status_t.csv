model,first_success,last_success
a,2020-01-01 00:00:00,2020-01-02 00:00:00
b,2020-01-02 00:00:00,2020-01-03 00:00:00
c,2020-01-03 00:00:00,2020-01-04 00:00:00
d,2020-01-01 00:00:00,2020-01-02 00:00:00
