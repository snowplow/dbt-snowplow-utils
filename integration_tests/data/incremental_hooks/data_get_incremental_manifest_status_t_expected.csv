test_case,min_first_success,max_first_success,min_last_success,max_last_success,models_matched_from_manifest,sync_count,has_matched_all_models
all model_in_run exist in manifest,2020-01-01 00:00:00,2020-01-03 00:00:00,2020-01-02 00:00:00,2020-01-04 00:00:00,3,3,true
some model_in_run exist in manifest,2020-01-01 00:00:00,2020-01-03 00:00:00,2020-01-02 00:00:00,2020-01-03 00:00:00,2,2,false
